server/routes/user.js

```diff
@@ -9,8 +9,7 @@ const { signUp, login, logout } = UserController;
 
 userRoute.post('/', validateUser, signUp);
 userRoute.post('/login', validateLogin, login);
-
-// logs out a user
+userRoute.get('/confirmEmail', UserController.confirmEmail);
```

Comment:
Hey @cvjude. Great job with the implementation. It's really awesome. I love the mail template. Great Job. I think you missed the `UserController.confirmEmail` line. I suggest you add it to the list of destructured items and call it accordingly.

Reply:
Thanks, how did I forget that. I'll get on it as soon as I can